src/sec_adapter_bundle.c

```diff
@@ -123,7 +123,8 @@ static Sec_Result Sec_StoreBundleData(Sec_ProcessorHandle* processorHandle, SEC_
             return SEC_RESULT_FAILURE;
         }
 
-        SecBundle_Delete(processorHandle, object_id);
+        // Note: Previously SA2 would delete existing bundles.  However, its unnecessary, and caused issues on certain targets.
```

Comment:
Let's spell out secapi2-adapter rather than rely on the SA2 abbreviation. Do we understand why this caused problems and why this is a reasonable approach?